api/registry/api.py

```diff
@@ -243,3 +262,50 @@ def get_score(request, address: str, community_id: int) -> ScoreResponse:
             exc_info=True,
         )
         raise InvalidScoreRequestException()
+
+
+class ScoreFilters(Schema):
+    community_id: int
+    limit: int = 50
+    offset: int = 0
+    addresses: List[str] = Field(None, alias="addresses")
+
+
+@router.get("/scores", auth=ApiKey(), response=List[ScoreResponse])
```

Comment:
Let us use the path `/score` not `/scores` (I think this is one of the general naming conventions).